src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -162,16 +165,18 @@ export function FileUploadViewPanel({setSubmitText, acceptList, acceptOneItem})
     return (
         <div style={{position: 'relative', height: '100%', display: 'flex', alignItems: 'stretch',
             flexDirection: 'column' }}>
+            <FileDropZone {...{setFileDrop, setDropEvent}} style={{height:'100%', width: '100%'}}>
                 <div className='FileUpload'>
                     <div className='FileUpload__input'>
                         <RadioGroupInputField
                             initialState={{value: uploadMethod[0].value}}
                             fieldKey={uploadOptions}
                             alignment={'horizontal'}
                             options={uploadMethod}
+                            isFileDrop={isFileDrop}
```

Comment:
I think you can change with and use an `useEffect` and `useFieldGroupValue` to change this field and not need extra properties.